---
title: Research
nav:
  order: 1
  tooltip: Research
---

# {% include icon.html icon="fa-solid fa-flask-vial" %}Research

Viruses are a burden to human health. Identifying host cell factors that bind and regulate viral RNA during the infection process is important for understanding how viruses can take over a host cell, subvert host processes and evade innate immune defense mechanisms. Insights into the underlying molecular interactions and mechanisms contribute towards the development of the next generation of rationally designed antivirals. 

We take an innovative RNA-centric approach that combines RNA interactomics techniques (e.g. RAP-MS, SHIFTR) with functional inference strategies (e.g. Perturb-seq) to identify and characterize novel pro- or antiviral mechanisms. Through the discovery and characterization of such mechanisms we hope to contribute to the development of the next generation of antivirals.

{%
  include figure.html
  image="images/mlab_research.png"
  caption="Graphical summary of the lab's research program Â©Sandy Westermann"
  link = "images/mlab_research.png"
  width="800px"
%}


<!-- ![](https://www.youtube.com/embed/c57mRJEIfWc) -->

## Highlights

How SARS-CoV-2 initiates the transcription and replication of its RNA genome is not fully understood. [In a recent publication in the journal Cell](https://doi.org/gstdzg), we show that the viral protein NSP9 primes the synthesis of both positive and negative-sense SARS-CoV-2 RNA. Unexpectedly, the host RNA-binding protein SND1 regulates this newly discovered protein-priming mechanism. For additional details, check out our video primer below.

<iframe width="632" height="356" src="https://www.youtube.com/embed/c57mRJEIfWc" title="Unexpected findings on SARS-CoV-2 replication" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe> 

## Current Focus Areas

**RNA Interactomics: A New Perspective on RNA Viruses**
The systematic analysis of RNA interactomes can uncover molecular signatures that detail the functional and regulatory dependencies of specific RNA molecules. Our lab examines these interaction signatures for different RNA viruses and their RNA products at various stages of the infection cycle to gain insights into the viral RNA replication program and its dependence on specific virus and host factors.

We are pioneering a new generation of RNA-centric tools that provide unprecedented insights into the molecular interactions between viruses and their hosts. One such tool, [SHIFTR](https://doi.org/10.1093/nar/gkae038), promises to revolutionize our understanding of viral RNA genomes, their design principles, and molecular building blocks. SHIFTR allows us to analyze interactions for any cis-regulatory element encoded by any virus, in its native state, and throughout the infection cycle. In doing so, we are unraveling the molecular blueprint that dictates the molecular workings of an RNA virus

{%
  include figure.html
  image="images/mlab_res_Fig1.png"
  caption="SHIFTR: a revolutionary tool for molecular virology"
  link = "images/mlab_res_Fig1.png"
  width="800px"
%}

Current projects in this area aim at developing technologies to assess RNA-protein interactions and their functional consequences at the single-cell and single-molecule level.

**CRISPR meets Single Cells: From Interaction to Function**
Complementing the area of RNA interactomics, we combine pooled CRISPR screens with single-cell transcriptomics (aka Perturb-seq) to systematically determine the impact of a large number of host factors on the gene expression patterns of both virus and host. Leveraging AI and machine learning, these single-cell readouts help identify pathways and programs relevant to viral infections and devise strategies to interfere with viral replication. Beyond single-cell sequencing, we employ advanced imaging techniques to obtain high-resolution views of cellular perturbations, crucial for defining infection outcomes in live cells.

{%
  include figure.html
  image="images/mlab_res_Fig2.png"
  caption="Perturb-seq: targeting RNA-modifying enzymes during Influenza A Virus (IAV) infection"
  link = "images/mlab_res_Fig2.png"
  width="800px"
%}

**Molecular Mechanisms: a Path to New Antivirals**
Our ultimate goal is to characterize the molecular mechanisms underlying (i) the successful execution of viral replication programs in human cells and (ii) the ability of a host cell to restrict viral pathogens. Using our unique RNA-centric approach, we profile interactions at unprecedented resolution and resolve functional consequences for each viral RNA type generated during infection. This high-resolution view allows us to distinguish between functionally diverse RNA populations, such as new and old RNA, incoming and replicated viral genomes, and positive and negative-strand viral RNA. This approach has uncovered fundamental aspects of viral RNA biology, including a previously overlooked protein-priming mechanism utilized by SARS-CoV-2, which is unexpectedly regulated by a host-encoded RNA-binding protein [Read more in Cell](https://doi.org/gstdzg).

{%
  include figure.html
  image="images/mlab_res_Fig3.png"
  caption="Effect of SND1 on NSP9-mediated priming of viral RNA synthesis"
  link = "images/mlab_res_Fig3.png"
  width="800px"
%}

**RNA Therapeutics and the Innate Immune System**
RNA-based therapeutics hold tremendous potential to revolutionize the treatment of human diseases, including infectious diseases. Our lab has established a technology unit dedicated to the design, synthesis, and purification of therapeutic mRNAs and the formulation of mRNA delivery vehicles, such as lipid nanoparticles. To enhance the design and biological activity of future RNA-based therapeutics, we investigate how therapeutic RNAs interact with various factors within target cells. We focus particularly on understanding the features of therapeutic RNAs that determine their stability and translational activity, as well as their interaction with the innate immune system. In this context, we are uniquely positioned to explore how the presence or absence of base modifications in therapeutic RNAs impacts their interaction profiles and influences the activation of innate immune pathways.


## Funding

Research in our lab is generously supported by the following sources and programs:

{% capture col1 %}
{%
  include figure.html
  image="images/logos/LOGO_ERC-FLAG_FP.png"
  caption=""
  link = "images/logos/LOGO_ERC-FLAG_FP.png"
  width="300px"
%}
{% endcapture %}

{% capture col2 %}
{%
  include figure.html
  image="images/logos/LOEWE.jpeg"
  caption=""
  link = "images/logos/LOEWE.jpeg"
  width="300px"
%}
{% endcapture %}

{% capture col3 %}
{%
  include figure.html
  image="images/logos/Logo_Pitzer_Stiftung.jpeg"
  caption=""
  link = "images/logos/Logo_Pitzer_Stiftung.jpeg"
  width="300px"
%}
{% endcapture %}

{% capture col4 %}
{%
  include figure.html
  image="images/logos/HFSP_horizontal_blue.jpeg"
  caption=""
  link = "images/logos/HFSP_horizontal_blue.jpeg"
  width="300px"
%}
{% endcapture %}

{% capture col5 %}
{%
  include figure.html
  image="images/logos/ForCovid_Logo.png"
  caption=""
  link = "images/logos/ForCovid_Logo.png"
  width="300px"
%}
{% endcapture %}

{% capture col6 %}
{%
  include figure.html
  image="images/logos/SPRIND.jpeg"
  caption=""
  link = "images/logos/SPRIND.jpeg"
  width="300px"
%}
{% endcapture %}

{% capture col7 %}
{%
  include figure.html
  image="images/logos/EMBO_logo.png"
  caption=""
  link = "images/logos/EMBO_logo.png"
  width="300px"
%}
{% endcapture %}

{% capture col8 %}
{%
  include figure.html
  image="images/logos/Helmholtz_Association_Logo.png"
  caption=""
  link = "images/logos/Helmholtz_Association_Logo.png"
  width="300px"
%}
{% endcapture %}

{%
  include cols.html
  col1=col1
  col2=col2
  col3=col3
%}

{%
  include cols.html
  col1=col4
  col2=col5
  col3=col6
%}

{%
  include cols.html
  col1=col7
  col2=col8
%}
